flink-table-store-core/src/main/java/org/apache/flink/table/store/file/schema/SchemaManager.java

```diff
@@ -157,6 +159,33 @@ public TableSchema commitNewVersion(UpdateSchema updateSchema) throws Exception
         }
     }
 
+    private void validatePrimaryKeysType(UpdateSchema updateSchema, List<String> primaryKeys) {
+        if (!primaryKeys.isEmpty()) {
+            Map<String, RowType.RowField> rowFields = new HashMap<>();
+            for (RowType.RowField rowField : updateSchema.rowType().getFields()) {
+                rowFields.put(StringUtils.lowerCase(rowField.getName()), rowField);
```

Comment:
Why use `lowerCase`?

Reply:
I will remove `lowCase` here.

But in fact there's a problem of the up/lowCase of table and column names. 

The ddl syntax between table store and spark/hive are different. The table and column name of spark/hive are case-insensitive, while in table store with flink, the table name is case-insensitive and column name is case-sensitive.

I think we need to verify the duplicate table/column names in the table store to be case-insensitive to avoid confusion for the users.  Then we can also use lowercase uniformly for comparison when doing some additional verification. What do you think?

Reply:
I created a jira for hive https://issues.apache.org/jira/browse/FLINK-29988

Reply:
I will try to fix it later

> I created a jira for hive https://issues.apache.org/jira/browse/FLINK-29988